### DISCUSSION TITLE
Ruby support in CI

### DISCUSSION THREAD

**Discussion by waghanza** [vcr/vcr#894] [Show and tell]
Hello,

I can see that CI runs on ruby **2.4**
https://github.com/vcr/vcr/blob/fa906679a38117bd369a1bdd86fb001f8af88bdc/.github/workflows/test.yml#L19

Maybe it could be accurate to remove this version, since **2.4** has reach EOL

https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/

Also, we can add **3.0** support

Regards,

**Comment by olleolleolle** [comment]
Thanks, this has been handled - we now include 3.0 and 2.4 is gone from the matrix.
